newsfragments/37638.significant.rst

```diff
@@ -0,0 +1,4 @@
+Replaced test_should_respond_400_on_invalid_request with test_ignore_read_only_fields in the test_dag_endpoint.py.
```

Comment:
TODO: We need to add a few more notes likely here and change the newsfragment number to reflect the PR number.